src/hooks/useInitializeLocationState.js

```
@@ -0,0 +1,25 @@
+import { useLocation } from 'react-router-dom'
+import useFeedStore from '@stores/useFeedStore'
+
+const useInitializeLocationState = () => {
+  const location = useLocation()
+  const resetSelectedField = useFeedStore((state) => state.resetSelectedField)
+
+  useEffect(() => {
+    const deleteDataState = (state) => {
+      if (!state?.usr) return state
+      const newState = { ...state, usr: { ...state.usr } }
+      if ('data' in newState.usr) delete newState.usr.data
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)